crates/tabby/src/main.rs

```diff
@@ -1,22 +1,23 @@
-mod routes;
-mod services;
-
-mod download;
-mod serve;
-
-#[cfg(feature = "ee")]
-mod worker;
-
 use clap::{Parser, Subcommand};
 use opentelemetry::{
     global,
     sdk::{propagation::TraceContextPropagator, trace, trace::Sampler, Resource},
     KeyValue,
 };
 use opentelemetry_otlp::WithExportConfig;
-use tabby_common::config::Config;
 use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt, EnvFilter, Layer};
 
+use tabby_common::config::Config;
```

Comment:
This seems to be from your rustfmt config. rustfmt always seems to change it back if I try to commit it:
![image](https://github.com/TabbyML/tabby/assets/26320625/39e8199f-5d3a-4c79-b095-6ddd5236f53d)
![image](https://github.com/TabbyML/tabby/assets/26320625/ad660626-b208-4eab-98ef-64b0c5b97b12)

Reply:
I believe the issue arises from the difference in configuration for rustfmt. Nonetheless, I would prefer to keep the formatting change as a separate and isolated modification.

Reply:
Well the configuration is set from the `.rustfmt.yaml` file in the repo, so it would be the same. I'll turn it off for now, but that of course means I can't guarantee that the code is properly formatted.

Reply:
Okay figured out that your config requires that you have rustfmt nightly, so now it should be better